.github/workflows/build.yml

```diff
@@ -226,7 +226,7 @@ jobs:
      - name: Cygwin git configuration
        shell: bash
        # workaround for https://github.com/actions/checkout/issues/760
-       run: mkdir "$HOME" && git config --system core.autocrlf false && git config --global --add safe.directory "${GITHUB_WORKSPACE}"
+       run: mkdir "$HOME" && git config --system core.autocrlf false && git config --global --add safe.directory "/cygdrive/d/a/temurin-build/temurin-build"
```

Comment:
Is that always going to be true?

Reply:
it is running in windows so the path should be correct. 
but it is just a workaround till they have it fixed formally in GH Action, then can have line reverted